src/model/repositories/AuthRepository.php

```diff
@@ -57,57 +53,59 @@ public function createUserToExistingEmail(User $user): ?User {
     
         try {
     
-            $wasUpdated = $query->execute([
```

Comment:
How come we got rid of the htmlspecialchars?

Reply:
Input sdhould be already sanitized in the controller :)